docs/debugging.md

```diff
@@ -56,13 +56,13 @@ It's easy to view the current Debugger state in the console with
 the `appStore` global. Because the data is immutable, you'll need to request the state each time you want to see it.
 
 ```js
-appStore.getState().sources.toJS()
+appStore.getState().sources.toJS();
```

Comment:
Am I wrong but would this be `dbg.appStore` now or does it remain a separate global?

Reply:
ahh - we should drop appStore and just use `dbg.store`